python/tvm/generic.py

```diff
@@ -0,0 +1,18 @@
+"""Generic opertor interface for overloading."""
+# pylint: disable=unused-argument
+
+def add(lhs, rhs):
+    """generic add interface"""
+    raise NotImplementedError
+
+def sub(lhs, rhs):
```

Comment:
Let us make the function name compatible with numpy, which means this should be substract